test/navbar-components/navigationBarTest.js

```diff
@@ -379,10 +379,8 @@ describe('navigationBar tests', function () {
           this.app.client
             .activateURLMode()
             .waitForExist(urlBarIcon)
-
-            // TODO: Fix
             .getAttribute(urlBarIcon, 'class').then((classes) =>
-              classes.includes('fa-unlock') && classes.includes('insecure-color')
+              classes.includes('fa-unlock')
```

Comment:
you can't have two data attributes with the same name, so it would be something like this `[data-test-id="fa-unlock"][data-test2-id="insecure-color"] and yes we can always do that